integrated_channels/canvas/client.py

```diff
@@ -349,6 +349,33 @@ def cleanup_duplicate_assignment_records(self, courses):
 
         return status_code, message
 
+    def update_participation_types(self, canvas_pk):
```

Comment:
If you wanted to be _**extra**_  thorough you could write a test for this method, but I realizing we want to get this out and I don't think it's overly complicated so you can skip for now.